.github/workflows/ci-base-tests.yml

```diff
@@ -54,4 +54,59 @@ jobs:
             --ignore=./smarts/core/tests/test_smarts_memory_growth.py \
             --ignore=./smarts/env/tests/test_benchmark.py \
             --ignore=./smarts/env/tests/test_learning.py \
-            -k 'not test_long_determinism'
\ No newline at end of file
+            -k 'not test_long_determinism'
+
+  mac:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
+    strategy:
+      matrix:
+        tests:
+          - ./envision
+          - ./smarts/contrib
+          - ./tests
+          - ./smarts/sstudio --ignore=./smarts/sstudio/genhistories.py
```

Comment:
Why ignore `--ignore=./smarts/sstudio/genhistories.py` a non test file in mac test?

Reply:
Its because of the Waymo import in `genhistories.py`. Pip package provided by Waymo for the dataset is only for Linux. You need to compile your own packages for other OS. So I ignored the file here since it checks for dependencies for all files in `sstudio` folder.

Reply:
I can also put the import statement in a try block if that's a better solution.